src/app/components/shell/lower-sticky-note/lsn-content.js

```diff
@@ -23,7 +23,7 @@ function NoteWithImage({bannerInfo}) {
 
 function NoteWithoutImage({bannerInfo}) {
     return (
-        <div className="content">
+        <div className="content" role='marquee' aria-label='sticky note'>
```

Comment:
Would a `status` role be more appropriate here, because the banner text doesn't (seem to) change? I'm not sure what is normal to use here, but the marquee docs suggest it should be for live, updating content.

Reply:
I had a hard time picking. Both status and marquee are live regions. It seemed more of an announcement than a status message to me.

Reply:
Dialog was the other contender.

Reply:
Yeah that's a good point about live region, which it doesn't seem to be. `dialog` now seems like the most appropriate candidate to me.